src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -68,7 +74,41 @@ public String deleteArticle(String articleId, String userId, HttpSession session
         if (!userId.equals(session.getAttribute("userId"))) {
             throw new IllegalArgumentException("다른 사람의 게시글은 삭제할 수 없습니다.");
         }
+        if (replyService.findReplyAllByArticleId(Integer.parseInt(articleId)).size()
+                != replyService.findReplyByArticleIdAndUserId(Integer.parseInt(articleId), userId).size()) {
+            throw new IllegalArgumentException("다른 사람의 댓글이 있는 게시물은 삭제할 수 없습니다.");
+        }
         articleService.deleteArticle(Integer.parseInt(articleId));
```

Comment:
```suggestion
        articleService.deleteArticle(articleId);                
```
메소드 시그니처에서 처음부터 int로 받으면 되지 않나요?

Reply:
참 그리고 id는 관행상 Long 을 사용하는 편입니다.

Reply:
게시글을 삭제하면 댓글도 같이 삭제해야 좋지 않을까요?